src/components/books/books.jsx

```diff
@@ -5,8 +5,8 @@ import Book from './book';
 import Header from '../header';
 
 const books = () => (
-  <div className='main-container'>
-    <div className='container'>
+  <div className='main-container sm:w-full h-screen'>
```

Comment:
- [ ] I got a blank page when I start your application. Please take a look at this issue.

See test demo|
:-----------------:|
![a1](https://user-images.githubusercontent.com/39523312/161392518-6db2f3b7-1f26-4540-ae9e-be2cffa9cf8c.gif)

Reply:
Hello @mmsesay. I don't really know why it does not work for you. please install all the needed libraries and check again, please try inspecting your browser to know why it does not work for you because it works very well in localhost and live server. If you are not sure if it works or not please check again my live demo [here](https://book-store-mwa.netlify.app/)
Please watch my recording in this link to see how my project works [My recording to explain how the project works locally](https://www.awesomescreenshot.com/video/8201324?key=4859a2f38f4e8e7c833eb94e125223dd)

Reply:
Hello @mwafrika, I have checked but it's still the same on my end. I even checked the live version and but got the same result.

Reply:
let me share my screen now

Reply:
may be you have some problem with your tools or internet